package-res/ccc/core/base/chart/chart.axes.js

```diff
@@ -69,6 +69,20 @@ pvc.BaseChart
     },
 
     _initAxes: function(hasMultiRole) {
+        
+        // CDF603 C
+        // Get axis state 
+        // The state is to be kept between render calls
+        var axesState, 
+            oldByType = def.copy( {}, this.axesByType );
```

Comment:
No need to copy. The whole "index" object-tree is replaced by a new one in the lines following the added ones.
```javascript
prevAxesByType = this.axesByType;
```